### PR TITLE
Meeting notes: 30.03.2023 Polyculture

### PR DESCRIPTION
**Pull request by Benjaminimal**
Here are the notes from today's polyculture meeting.

@markus2330
I created a sheet to gather relationships consistently.
It's linked in the meeting notes.
Please have a look at it.
If you think we can work with this I'll forward it to the other people.

I tried to minimize work when importing this and make adding or changing data as easy as possible.
To reduce duplication, spelling, casing, and whitespace errors I made a "Plants" sheet with full taxonomic information.
When adding relations all you have to do is select values from drop downs.
The currently entered data are just examples and of course not any real relationships.

## Basics

- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.
- [x] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

File: doc/meetings/2023_03_30_polyculture.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+- Plants for a future and Natural capital plant database have relationship info readily available in spreadsheet format.
+  We can probably use the info if we don't use any of relationship details.
+- Playing around with chatGPT could be an option.
+  They quality might not be best here.
+  Can we automate these queries or do we have to run each manually?
+
+### Decisions
+
+1. Uliana will start research on plant relationships.  
+   Frist the quality of the following datasources is be assessed.
```
Comment (yvonnemarkl) [inline doc/meetings/2023_03_30_polyculture.md:76]:
```suggestion
   First the quality of the following datasources has to be assessed.
```

**Review by markus2330 — CHANGES_REQUESTED** [review]
Thank you for writing the meeting notes, great job to summarize so much content!

File: doc/meetings/2023_03_30_polyculture.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+1. Uliana will start research on plant relationships.  
+   First the quality of the following datasources has to be assessed.
+
+   - [Plants for a Future](https://pfaf.org/)
+   - [Natural Capital Plant Database](https://permacultureplantdata.com/)
+   - [Permaculture News](https://www.permaculturenews.org/2010/07/30/companion-planting-guide/)
+
+   Then a collection of relationships in the following format is created.  
+   You can find the document [here](https://docs.google.com/spreadsheets/d/1jtsng8H4SkwK2gxK_qF1XkC0ygpjXZgMUKKj1x3TA9M/).
```
[the remark this answers is not in the document]
Comment (Benjaminimal) [reply doc/meetings/2023_03_30_polyculture.md:82]: Added a license.

Yes, the examples are not correct.
> The currently entered data are just examples and of course not any real relationships.

File: doc/meetings/2023_03_30_polyculture.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+   - Left side
+     - Common name
+     - Variety
+     - Species
+     - Genus
+     - Subfamily
+     - Family
```
[the remark this answers is not in the document]
Comment (Benjaminimal) [reply doc/meetings/2023_03_30_polyculture.md:90]: Those are the names of the ranks the plant belongs to.
"Taxonomic rank" describes on which level the relationship applies.

File: doc/meetings/2023_03_30_polyculture.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+     - low
+     - medium
+     - high
+     - approved
+   - Relationship type
+     - companion
+     - antagonist
+     - neutral
+   - Notes
+   - Sources
```
[the remark this answers is not in the document]
Comment (Benjaminimal) [reply doc/meetings/2023_03_30_polyculture.md:109]: She can also add columns if she needs to.

File: doc/meetings/2023_03_30_polyculture.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+   First the quality of the following datasources has to be assessed.
+
+   - [Plants for a Future](https://pfaf.org/)
+   - [Natural Capital Plant Database](https://permacultureplantdata.com/)
+   - [Permaculture News](https://www.permaculturenews.org/2010/07/30/companion-planting-guide/)
+
+   Then a collection of relationships in the following format is created.  
+   You can find the document [here](https://docs.google.com/spreadsheets/d/1jtsng8H4SkwK2gxK_qF1XkC0ygpjXZgMUKKj1x3TA9M/).
+
+   - Left side
```
[the remark this answers is not in the document]
Comment (Benjaminimal) [reply doc/meetings/2023_03_30_polyculture.md:86]: See https://github.com/ElektraInitiative/PermaplanT/pull/186/commits/5eecf4782900f6a945cead393351a26da120ab8f

**Pull request merged**